composer.json

```diff
@@ -42,6 +44,9 @@
         "symfony/form" : "Needed to be able to use the AuthorizeFormType",
         "symfony/console": "Needed to be able to use commands"
     },
+    "provide": {
+        "ext-mongo": "^1.6.14"
```

Comment:
see https://github.com/api-platform/core/pull/1293/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780

Reply:
If you're providing `ext-mongo`, you also have to install a package that provides the API (see https://github.com/api-platform/core/pull/1293/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780R28).

Reply:
To follow up on our conversation at the hack day, it is smarter to not install require the adapter in general, but rather on demand in travis.yml. One example can be found in the MongoDB ODM repository: https://github.com/doctrine/mongodb-odm/blob/cc122d144f32d661be853bf5af0a44acd63602a7/.travis.yml#L42..L44

Reply:
Thanks, I'll apply changes within next few days. Btw. according to our discussion in Cluj, have you had a chance to test my changes against your app and testsuites?

Reply:
I've hit a brick wall trying to update to Symfony 3.4. I'll run it against 3.3 to be sure ;)